cypress/pageFunction/HomePageFunctions.ts

```diff
@@ -0,0 +1,26 @@
+import { HomePageLocators } from '../locators/HomePageLocators';
+import { PageTextValues } from '../pageEnums/PageTextValues';
+/*This function basically performs the Homepage component validations*/
+export class HomePageFunctions{
+    static validateTitle(expectedTitle:string){
+        cy.title().should('eq',expectedTitle);
+    }
```

Comment:
Do we need a function for this ? Can be used directly !

Reply:
@ajinkyaces yeah we don't need function for this but for avoiding assertion in test file that's why I created a function